### DISCUSSION TITLE
链上运维平台建设实践

### DISCUSSION THREAD

**Discussion by Naturefir** [filecoin-project/community#268] [News and Events]
星际运维社区是基于Filecoin的技术社区，技术驱动、不断探索。每周会有线上/线下主题分享，欢迎各位行业大咖作客分享，也同样欢迎带着疑问共同探讨。安全稳定是Filecoin网络稳定收益的基石，邀请大家一起推动Filecoin生态发展，共同构建安全稳定的Web 3.0基础设施。

### 本期嘉宾：
星际联盟联合创始人&CTO  毛必盛

### 分享主题：
《链上运维平台建设实践》

### 分享时间：
2021年8月25号 19:00-20:00

### 线上直播地址：
http://live.bilibili.com/23317958

### 往期直播分享回顾：
https://space.bilibili.com/1326910011
![0826主题海报](https://user-images.githubusercontent.com/79625747/130763372-562ecf4b-8f45-4f91-bb90-5b02d40f322e.png)

**Comment by Naturefir** [comment]
欢迎大家积极参与，可以加入活动群
![二维码 - 客服小包](https://user-images.githubusercontent.com/79625747/130763736-aa9d087c-d212-4fcf-9711-0328a6cb0c07.png)

**Comment by Naturefir** [comment]
针对这个话题，大家如果有疑问或者想讨论的，欢迎留言讨论

**Comment by dcshy** [comment]
支持
专注Filecoin，自有技术团队近百人，t3+IDC机房，日封装能力2p+，B端独立节点集群方案。可出海，微信：IPFS-Shy
